STEERING-COMMITTEE.md

```diff
@@ -236,19 +244,20 @@ found in the [elections folder](./elections).
 
 ### Limitations on Company Representation
 
-No more than two seats may be held by employees of the same organization (or
-conglomerate, in the case of companies owning each other). Additionally, each
-end user seat (when available) must belong to an employee from a different
-organization. If the results of an election result in greater than two employees
-of the same organization, the lowest vote getters from any particular employer
-will be removed until representation on the committee is down to two.
-
-If employers change because of job changes, acquisitions, or other events, in a
-way that would yield more than 2 seats being held by employees of the same
-organization, sufficient members of the committee must resign until only two
-employees of the same employer are left. If it is impossible to find sufficient
-members to resign, all employees of that organization will be removed and new
-special elections held. In the event of a question of company membership (for
+No more than two contributing seats may be held by employees of the same organization
+(or conglomerate, in the case of companies owning each other). Additionally, each end
+user seat (when available) must belong to an employee from a different
+organization. No company may hold a majority (fifty percent or more) of the total
+Steering Committee seats through any combination of contributing, end-user, and
+selected seats. If the results of an election result in greater than two employees of
```

Comment:
```suggestion
Steering Committee seats. If the results of an election result in greater than two employees of
```

I would just leave out the "any combination of contributing, end-user, and selected seats" as I think this is redundant and a bit confusing (what is a combination of seats, anyway?).

Reply:
This is just making clear that the sum total of seats can't be a majority.  This means that the following would be disallowed:

```
2 $Foo elected seats
1 $Foo end-user seat
2 Non-$Foo elected seats
1 Selected seat (non-$Foo)
```
If $Foo is an end-user company.  Another combination that would be disallowed:
```
2 $Foo elected seats
2 Non-$Foo elected seats
1 Non-$Foo end-user seat
1 $Foo selected seat
```
(Because that would give $Foo a total of 50% of the committee representation)

Reply:
Whoops, we actually prevent the first scenario.  But, to your point, we should move most of the company representation limits to a section that's not under "Election Procedure"